js/modal.js

```diff
@@ -14,12 +14,14 @@
   // ======================
 
   var Modal = function (element, options) {
-    this.options        = options
-    this.$body          = $(document.body)
-    this.$element       = $(element)
-    this.$backdrop      =
-    this.isShown        = null
-    this.scrollbarWidth = 0
+    this.options             = options
+    this.$body               = $(document.body)
+    this.$element            = $(element)
+    this.$dialog             = this.$element.find('.modal-dialog')
+    this.$backdrop           =
```

Comment:
I think we can change that in a future PR for all files. It's especially hard to see for people who might not quite understand who ASI works and that this is legit.